tests/v3_api/scripts/pipeline/Jenkinsfile_ontag_matrix

```diff
@@ -42,6 +42,9 @@ if ( true == via_webhook() && (!(rancher_version ==~ rancher_version_regex)) ) {
   if (rancher_version.startsWith("v2.2") || rancher_version == "master") {
     branch = "master"
   }
+  if (env.BRANCH) {
```

Comment:
This pipeline is used by both the manual and ontag jobs. If we don't allow the override of branch here, the branch will always be set to master when running manually.